### DISCUSSION TITLE
KiBilgi

### DISCUSSION THREAD

**Discussion by Ceremenli53** [community/community#31311] [New to GitHub] [answered]
Merhaba ben yeni üye oldum ama pek anlamiyorum bana yardımcı olursanız sevinirim

**Comment by queenofcorgis** [comment, marked as answer]
Welcome to GitHub and Community on Discussions. To get started I recommend checking our [New User Checklist](https://github.com/orgs/community/discussions/27367) and [GitHub Skills](https://skills.github.com/).

Then let the community know if you have any questions 🚀 .
